content/java/concepts/math-methods/terms/floor/floor.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Floor'
+Description: 'Returns the largest integer value that is less than or equal to the argument. When the provided value is either an integer, zero, not a number, or positive or negative infinity.'
+Subjects: 
+    -'Code Foundations'
+    -'Computer Science'
+Tags: 
+    - 'Documentation'
+CatalogContent: 
+  - 'learn-java/static-methods-of-the-math-class'
```

Comment:
```suggestion
  - 'learn-java'
  -  'paths/computer-science'
```

Just the parent directory of the free course slug is sufficient. Let's also include a paid path/course slug. 😄

Reply:
Implemented suggested change.